rendermodules/rough_align/schemas.py

```diff
@@ -1,12 +1,113 @@
 from argschema import InputDir
 import marshmallow as mm
-from argschema.fields import Bool, Float, Int, Nested, Str, InputFile, List
+from argschema.fields import (
+        Bool, Float, Int, Nested, OutputDir,
+        Str, InputFile, List, Dict)
 from argschema.schemas import DefaultSchema
-from ..module.schemas import RenderParameters, OutputStackParameters
-from marshmallow import post_load, ValidationError, pre_load
+from ..module.schemas import (
+        RenderParameters,
+        StackTransitionParameters,
+        ProcessPoolParameters)
+from marshmallow import post_load, ValidationError
 import argschema
 
 
+# class RoughSlicerSchema(InputStackParameters, ProcessPoolParameters):
+#     x_slice_locs = List(
+#         Float,
+#         required=False,
+#         default=[0.5],
+#         cli_as_single_argument=True,
+#         description="fractional location of x_slices")
+#     y_slice_locs = List(
+#         Float,
+#         required=False,
+#         default=[0.5],
+#         cli_as_single_argument=True,
+#         description="fractional location of x_slices")
+#     slice_locs_as_pixels = Bool(
+#         required=False,
+#         default=False,
+#         missing=False,
+#         description=("interpret slice_locs as absolute pixel "
+#                      "coordinates instead of fractional."))
+#     output_dir = OutputDir(
+#         required=True,
+#         description="Location_of_output_slices")
+#     bounds_from_stack = Str(
+#         require=False,
+#         default=None,
+#         missing=None,
+#         description=("get bounds from this stack instead of input_stack"))
+
+
+class MakeAnchorStackSchema(StackTransitionParameters):
+    transform_json = InputFile(
+        required=True,
+        description=("Human generated list of transforms."
+                     "Keys are of form <z>_*.png where z matches "
+                     "a tilespec in input_stack and values are "
+                     "AffineModel transform jsons"))
+    zValues = List(
```

Comment:
This should already be in the StackTransitionParameters (automatically generated based on `minZ` and `maxZ` if only inputting those)

Reply:
I wanted to use the stack transition module, but, there's no meaning to minZ and maxZ relative to the input stack in this case, so I needed something here to keep the parent class happy.

Reply:
I see, I didn't think about the use case with an external specification of zs.  Going to resolve this, but it's likely a quick modification of the `ZValueParameters` to support this.